.github/workflows/build.yml

```diff
@@ -14,16 +14,25 @@ jobs:
         os: [windows-latest, macOS-latest, ubuntu-latest]
     steps:
       - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
       - name: Install Rust Nightly Toolchain
         uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly-2019-11-04
           override: true
-      - name: Run cargo check
+      - name: Install binaryen
+        run: npm install binaryen -g
```

Comment:
I would like to move this responsibility to the `minimize_wasm.sh` script. It should install binaryen if it was not installed (of course not globally) and then run it. `minimize_wasm` is meant to be useful not only on CI, it sohuld work locally, and requiring binaryen to be installed locally or on CI is a bad assumption. So I believe this step should not be a CI step - it should be done in the script.

Reply:
I just moved the responsibility for installing binaryen to `minimize_wasm.sh`